desktop/lib/window-manager/index.js

```diff
@@ -79,6 +79,13 @@ function openWindow( windowName ) {
 			windows[windowName].handle.on( 'closed', function() {
 				windows[windowName].handle = null;
 			} );
+
+			if ( Config[settings.config].wpDragAndDropDisabled ) {
+				windows[windowName].handle.webContents.on( 'will-navigate', function( event ) {
```

Comment:
@roundhill thanks for taking care of this 👍 

I just need to mention this for the record: This will not just disable drag and drop as it will disable all non-hash based navigation. While I don't think that this is a big deal for the windows that are currently affected, I believe we should add a note to avoid endless debugging in the future.